plugins/importexport/datacite/DataciteInfoSender.inc.php

```diff
@@ -137,8 +141,10 @@ function _getJournals() {
 			if ($doiPrefix) {
 				$journals[] =& $journal;
 			} else {
-				$this->notify(SCHEDULED_TASK_MESSAGE_TYPE_WARNING,
-					__('plugins.importexport.crossref.senderTask.warning.noDOIprefix', array('path' => $journal->getPath())));
+				$this->addExecutionLogEntry(
```

Comment:
You're probably right.  I only looked at the function name and thought it might be shared by the UI.  The parent class makes that unlikely.